src/panels/config/ha-panel-config.ts

```diff
@@ -213,6 +213,15 @@ export const configSections: { [name: string]: PageNavigation[] } = {
       iconColor: "#616161",
     },
   ],
+  application_credentials: [
+    {
+      component: "application_credentials",
+      path: "/config/application_credentials",
+      translationKey: "ui.panel.config.application_credentials.caption",
+      iconPath: mdiDevices,
```

Comment:
This seems like the wrong icon for this page, as we already use it for `devices`

Reply:
The icon isn’t used here. Is it okay to remove it?

Reply:
If the typing allows you to, yes